packages/functions/src/delete.ts

```diff
@@ -18,3 +19,17 @@ export const main = Util.handler(async (event) => {
 
   return JSON.stringify({ status: true });
 });
+
+export const main2 = Util.handler(async (event) => {
+  const params = {
+    TableName: Resource.Notes.name,
+    Key: {
+      userId: event.requestContext.authorizer?.iam.cognitoIdentity.identityId,
+      noteId: event?.pathParameters?.id, // The id of the note from the path
+    },
+  };
```

Comment:
testing multi-line fix
```suggestion
  const params = {
    TableName: Resource.Notes.Name, // Add comment here
    Key: {
      userId: event.requestContext.authorizer?.iam.cognitoIdentity.identityId,
      noteId: event?.pathParameters?.id, // Update comment here
    },
  };
```